exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,13 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if(grade>=50){
+    return "passed";
+  }else{
+    return "failed";
+  }
+}
```

Comment:
yeah for just for learning code we can do if-else statement like this you did great job but in the coding we use else if any of our options are not true , then we give else to leave some messages like this: 
if (grade < 50) {
    return " failed";
  } else if (grade >= 50) {
    return " passed";
  } else {
    return " not a valid grade";
  }